### PR TITLE
[fix](doc) spell error fixes for FE & BE Config documents

### PR DESCRIPTION
**Pull request by Nitin-Kashyap**
Spell error fixes in documentation for FE & BE configurations page

### CODE REVIEW

File: docs/zh-CN/docs/admin-manual/config/be-config.md [hunk 1]
```diff
@@ -203,7 +203,7 @@ BE 重启后该配置将失效。如果想持久化修改结果，使用如下
 
 * 类型：int32
 * 描述：配置BE的所属于的集群id。
-  - 该值通常由FE通过心跳向BE下发，不需要额外进行配置。当确认某BE属于某一个确定的Drois集群时，可以进行配置，同时需要修改数据目录下的cluster_id文件，使二者相同。
+  - 该值通常由FE通过心跳向BE下发，不需要额外进行配置。当确认某BE属于某一个确定的Doris集群时，可以进行配置，同时需要修改数据目录下的cluster_id文件，使二者相同。
```
Comment (zy-kkk) [inline docs/zh-CN/docs/admin-manual/config/be-config.md:206]: Our usual norm is to add a space between Chinese and English

File: docs/zh-CN/docs/admin-manual/config/be-config.md [hunk 2]
```diff
@@ -224,7 +224,7 @@ BE 重启后该配置将失效。如果想持久化修改结果，使用如下
 
 #### `es_scroll_keepalive`
 
-* 描述：es scroll Keeplive保持时间，默认5分钟
+* 描述：es scroll keep-alive保持时间，默认5分钟
```
Comment (zy-kkk) [inline docs/zh-CN/docs/admin-manual/config/be-config.md:227]: ditto

File: docs/zh-CN/docs/admin-manual/config/be-config.md [hunk 3]
```diff
@@ -411,7 +411,7 @@ BE 重启后该配置将失效。如果想持久化修改结果，使用如下
 #### `enable_prefetch`
 
 * 类型：bool
-* 描述：当使用PartitionedHashTable进行聚合和join计算时，是否进行HashBuket的预取，推荐设置为true。
+* 描述：当使用PartitionedHashTable进行聚合和join计算时，是否进行HashBucket的预取，推荐设置为true。
```
Comment (zy-kkk) [inline docs/zh-CN/docs/admin-manual/config/be-config.md:414]: ditto

File: docs/zh-CN/docs/admin-manual/config/be-config.md (hunk 1 above)
Comment (Nitin-Kashyap) [reply docs/zh-CN/docs/admin-manual/config/be-config.md:206]: fixed

File: docs/zh-CN/docs/admin-manual/config/be-config.md (hunk 2 above)
Comment (Nitin-Kashyap) [reply docs/zh-CN/docs/admin-manual/config/be-config.md:227]: fixed

File: docs/zh-CN/docs/admin-manual/config/be-config.md (hunk 3 above)
Comment (Nitin-Kashyap) [reply docs/zh-CN/docs/admin-manual/config/be-config.md:414]: fixed

**Review by LemonLiTree — APPROVED** [review]
LGTM

**Comment by LemonLiTree** [comment]
GOOD JOB

**Review by zy-kkk — APPROVED** [review]
LGTM

**Comment by zy-kkk** [comment]
run buildall

**Pull request merged**